### PR TITLE
Keep the original file name in the cart

### PR DESCRIPTION
**Pull request by girishpanchal30**
### Summary
I've kept the original file name on the cart page with the unique hash. e.g: `<original file name>.<random hash>.<file extension>`

### Will affect visual aspect of the product

YES/NO

### Test instructions

-
-

## Check before Pull Request is ready:

* [ ] I have [written a test](CONTRIBUTING.md#writing-an-acceptance-test) and included it in this PR
* [x] I have [run all tests](CONTRIBUTING.md#run-tests) and they pass
* [x] The code passes when [running the PHP CodeSniffer](CONTRIBUTING.md#run-php-codesniffer)
* [x] Code meets [WordPress Coding Standards](CONTRIBUTING.md#coding-standards) for PHP, HTML, CSS and JS
* [x] [Security and Sanitization](CONTRIBUTING.md#security-and-sanitization) requirements have been followed
* [x] I have assigned a reviewer or two to review this PR (if you're not sure who to assign, we can do this step for you)

Closes https://github.com/Codeinwp/woocommerce-product-addon/issues/191

### CODE REVIEW

**Comment by AndreeaCristinaRadacina** [comment]
An error occurs when trying to upload a file, using this build:
image


It doesn't happen with the current production version.

**Comment by girishpanchal30** [comment]
@AndreeaCristinaRadacina Please make to disable the pro plugin or download it from production and check it.
It might be you have used a pro plugin from another PR build.

Thanks

**Comment by AndreeaCristinaRadacina** [comment]
@girishpanchal30, works as expected! Thank you!

File: inc/files.php [hunk 1]
```diff
@@ -522,6 +524,8 @@ function ppom_uploaded_file_preview( $file_name, $settings ) {
 // Trim long filename to short
 function ppom_files_trim_name( $file_name ) {
 
+	return $file_name;
```
Comment (Soare-Robert-Daniel) [inline inc/files.php:527]: Was this shortcircuit for debugging only?

Comment (girishpanchal30) [reply inc/files.php:527]: @Soare-Robert-Daniel  No, I think I missed removing the previous code, now we are displaying the full name.

I will remove it and push the changes

Comment (girishpanchal30) [reply inc/files.php:532]: @Soare-Robert-Daniel I've removed the code, please check the latest commit.

**Pull request merged**